src/pages/Dashboard/Event.tsx

```diff
@@ -0,0 +1,164 @@
+                }
+            });
+
+            if (!response.ok) {
+                throw new Error('Network response was not ok');
+            }
+            const data: eventType = await response.json();
+            setEvent({
+                ...data,
+                date: new Date(data.date),
+            });
+            console.log(data);
+        } catch (error) {
+            console.error('Error fetching event:', error);
+        } finally {
+            setLoading(false);
+        }
+    }
+
+    useEffect(() => {
+        checkMember();
+        fetchEvent();
+    }, [event_id]);
+
+    if (loading) return <p>Loading...</p>;
+    if (!event) return <p>No event details available.</p>;
+
+    return (
+        <div className="background-event">
+            <div className="header">
+                <div className="header-icon">
+                    <a href="/"><img src="./src/assets/pmc_logo.svg" className="logo" ></img></a>
+                </div>
+                <nav className="header-nav">
+                    <a href="#upcoming-events" className="header-link">Events</a>
+                    <div>
+                        {isLoggedIn ? (
+                            <a href="/profile" className="header-link">Profile</a>
+                        ) : (
+                            <a href="/" className="header-link">Profile</a>
+                        )}
+                    </div>
+                    {/* <a href="/" className="header-link">Profile</a> */}
+                </nav>
+            </div>
+
+
+
+            <div className="event-container">
+                <h2 className="event-title">{event.name}</h2>
+                <div className="event-details-container">
+                    <div className="event-details">
+                        <p>{event.date.toDateString()}</p>
+                        <p>{event.location}</p>
+                        <p>member price: ${event.member_price !== undefined ? event.member_price.toFixed(2) : "N/A"}</p>
+                        <p>non-member price: ${event.non_member_price !== undefined ? event.non_member_price.toFixed(2) : "N/A"}</p>
+                        {event.attendees ? (
+                            <div>
+                                <p>{event.attendees.length}/50 spots filled</p>
+                                {/* {event.attendees.length > 0 ? (
+                                    <ul>
+                                        {event.attendees.map((attendee) => (
+                                            <li key={attendee.id}>{attendee.name}</li>
+                                        ))}
+                                    </ul>
+                                ) : (
+                                    <p>no attendees in this event...</p>
+                                )} */}
+                            </div>
+                        ) : (
+                            <p>no attendees signed up yet...</p>
```

Comment:
Maybe we can change this to something optimistic? "Be the first to sign up!" or something lol

Reply:
lol I like this. Let's do it!